curvefs/src/client/main.c

```diff
@@ -64,23 +64,50 @@ void extra_options_help() {
     print_option_help(
         "fstype", "[required] type of filesystem to be mounted (s3/volume)");
     print_option_help("conf", "[required] path of config file");
-    print_option_help("mdsAddr", "mdsAddr of curvefs cluster");
+    printf("    --mdsAddr              mdsAddr of curvefs cluster\n");
+}
+
+char* parse_option_by_pattern(int* argc, char*** argv, const char* pattern) {
+    // parse option that fuse_opt_parse cannot parse
+    // remove it from argv, update argc
+    int newArgc = *argc;
+    char** newArgv = (char**)malloc(sizeof(char*) * newArgc);
```

Comment:
this memory is leaked, but it's acceptable.

Reply:
rewrite